linkcheck/director/console.py

```diff
@@ -64,7 +63,9 @@ def flush(self):
 def internal_error(out=stderr, etype=None, evalue=None, tb=None):
     """Print internal error message (output defaults to stderr)."""
     print(os.linesep, file=out)
-    print(_("""********** Oops, I did it again. *************
+    print(
+        _(
+            """********** Oops, I did it again. *************
```

Comment:
ugh. this also looks worse... maybe remove a few `****` instead?

Reply:
The `****` are in the output, so best leave those. I've changed it to go back to the original, that passes flake8.